test/spec/Feature/Query/RpcSpec.hs

```diff
@@ -1478,3 +1478,10 @@ spec actualPgVersion =
             "details":"DETAIL is missing in the RAISE statement",
             "hint":"DETAIL must be a JSON object with obligatory keys: 'status', 'headers' and optional key: 'status_text'."}|]
           { matchStatus = 500 }
+
+    context "test function temp_file_limit" $
+      let auth = authHeaderJWT "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoicG9zdGdyZXN0X3Rlc3Rfc3VwZXJ1c2VyIiwiaWQiOiJqZG9lIn0.LQ-qx0ArBnfkwQQhIHKF5cS-lzl0gnTPI8NLoPbL5Fg" in
```

Comment:
Hm, here you're using the `postgrest_test_superuser` right? Maybe add a comment so it's clearer?

---

Ideally we would have a function on the haskell tests, like this one on the io tests:

https://github.com/PostgREST/postgrest/blob/85a4e35a2de869ddf9b6c5f6893a3154767cbc76/test/io/test_io.py#L45

So we don't have to paste the JWT.. this could be another PR.